tests/toit/utils.toit

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2024 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+import host.pipe
+import system
+
+class ToitBin:
```

Comment:
I would call this `ToitBinary` or `ToitExecutable`.

Reply:
Done.